test/integration/app.js

```diff
@@ -17,6 +17,26 @@ function expect404Page(err, res) {
   expect(res.text).to.contain('Page not found');
 }
 
+function expectPeakPractice($) {
+  expect($('.local-header__title').text().trim())
+    .to.equal('Park Practice');
+  expect($('.gp-address').text().trim())
+    .to.equal('Eastbourne Park Primary Care Centre, Broadwater Way, Eastbourne, East Sussex, BN22 9PQ');
+  const contactDetailsTextFirst = $('.column--one-half:first-child p').first().text().trim();
+  const contactDetailsTextLast = $('.column--one-half:first-child p').last().text().trim();
+
+  expect(contactDetailsTextFirst).to.equal('Reception: 01323 502200');
+  expect(contactDetailsTextLast).to.equal('Fax: 01323 500527');
+  expect($('.gp-email').text()).to.equal('admin.parkpractice@nhs.net');
```

Comment:
I know these tests have only been moved but if I think about what the tests for the text are actually testing it seems to be the content. I'm wondering if this is the best place to test the content and whether it really needs testing at all. Testing the correct data gets rendered is fine and I know the content is partially doing that but it feels like it is duplicating the tests of the `href` (in this specific example at least).
What do you think?

Reply:
I think all of the tests that are asserting that GUI elements are rendering could all be done quicker and more robustly via UX tests with a stubbed service, with a handful of acceptance tests later on in the build pipeline. Maybe something to address another time?

Reply:
Agreed, issue #239 created.